e2e-tests/ukef/cypress/support/commands.js

```diff
@@ -1,8 +1,6 @@
 import 'cypress-file-upload';
 import './click-events';
 
-import './commands/click-events';
```

Comment:
Can we revert this please and the file deletions below, seems this will break some tests in the `ukef` E2E directory?